common-npm-packages/kubernetes-common/kubelogin.ts

```diff
@@ -63,9 +63,7 @@ export class Kubelogin {
 
       if (authType == 'spnCertificate') {
         taskLib.debug('certificate based endpoint');
-        let certificateContent: string = taskLib.getEndpointAuthorizationParameter(connectedService, 'servicePrincipalCertificate', false);
-        servicePrincipalKey = path.join(taskLib.getVariable('Agent.TempDirectory') || taskLib.getVariable('system.DefaultWorkingDirectory'), 'spnCert.pem');
-        fs.writeFileSync(servicePrincipalKey, certificateContent);
+        throw taskLib.loc('AuthSchemeNotSupported', `${authScheme} with a certificate`);

```

Comment:
Could you please add developer comment for clarification future development:

`// kubelogin doesn't support PEM format.`